node/cn/api.go

```diff
@@ -271,6 +273,32 @@ func (api *PrivateAdminAPI) GetSpamThrottlerCandidateList(ctx context.Context) (
 	return throttler.GetCandidates(), nil
 }
 
+func (api *PrivateAdminAPI) GetDebugAPILimiter(ctx context.Context) string {
+	if !api.cn.config.DisableUnsafeDebug {
+		return fmt.Sprintf("unsafe-debug is enabled")
+	}
+	return fmt.Sprintf("heavyAPILimit: %d, stateRegenerationTimeLimit: %.fs", tracers.HeavyAPIRequestLimit, api.cn.config.StateRegenerationTimeLimit.Seconds())
+}
+
+func (api *PrivateAdminAPI) SetDebugAPILimiter(ctx context.Context, heavyAPILimit int32, duration string) (bool, error) {
+	if !api.cn.config.DisableUnsafeDebug {
+		return false, errors.New("unsafe-debug is enabled")
+	}
+	timeLimit, err := time.ParseDuration(duration)
+	wrongDuration := err != nil || timeLimit <= time.Duration(0)
+
+	if heavyAPILimit <= 0 && wrongDuration {
```

Comment:
How about proceeding with the suitable value? Presently, the API requires updates for both values even if an operator intends to modify only one. Alternatively, you could consider splitting the function into two separate ones for intuitive.